docs/pages/develop/development-builds/expo-go-to-dev-build.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Convert from Expo Go to a development build
+sidebar_title: Expo Go to development build
+description: How to migrate your Expo Go project to use development builds
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+To migrate from Expo Go to a development build, you'll need to:
+
+<Step label="1">
+
+## Install the `expo-dev-client`
+
+The Expo Dev Client library includes the launcher UI (shown in the screenshots below), dev menu, extensions to test updates, and more. The Expo Go app has the dev menu built in, and that's why you need to install it separately for a development build.
```

Comment:
```suggestion
The Expo Dev Client library includes the launcher UI (shown in the screenshots below), dev menu, extensions to test over-the-air updates, and more. The Expo Go app has the dev menu built in, and that's why you need to install it separately for a development build.
```

Let's expand updates part to be specific about what updates are here in the context of a dev client and to avoid confusion.